gis/centreline/sql/create_matview_centreline_intersection_point_latest.sql

```diff
@@ -0,0 +1,19 @@
+CREATE MATERIALIZED VIEW gis_core.centreline_intersection_point_latest AS
+
+SELECT *
+FROM gis_core.centreline_intersection_point
```

Comment:
We should filter this to only include intersections that match centrelines we're pulling, to solve #819